soc/nordic/nrf53/nrf53_cpunet_mgmt.c

```diff
@@ -66,3 +68,33 @@ void nrf53_cpunet_enable(bool on)
 
 	__ASSERT_NO_MSG(ret >= 0);
 }
+
+#ifdef CONFIG_SOC_NRF53_CPUNET_ENABLE
+static int nrf53_cpunet_init(void)
```

Comment:
Looks quite similar to what we are doing with VPRs on 54s.
A common way to initialize and run secondary cores in Zephyr would be useful.

Reply:
agreed, a remoteproc-like API should be introduced. I'll look into this post-release.